### PR TITLE
LPS-202106 The web contents inside folder won't be shown in specific structure sheet

### PR DESCRIPTION
**Pull request by jkappler**
https://liferay.atlassian.net/browse/LPS-202106

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by jkappler** [comment]
@ruben-pulido, sending on top of BChans master because of conflicts

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -584,15 +584,20 @@ protected List<DropdownItem> getFilterNavigationDropdownItems() {
...
-		filterNavigationDropdownItems.add(
-			DropdownItemBuilder.putData(
-				"action", "openDDMStructuresSelector"
-			).setActive(
-				_journalDisplayContext.isNavigationStructure()
-			).setLabel(
-				LanguageUtil.get(httpServletRequest, "structures") +
-					StringPool.TRIPLE_PERIOD
-			).build());
+		if (FeatureFlagManagerUtil.isEnabled("LPS-194763") &&
```
Comment (ruben-pulido) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalManagementToolbarDisplayContext.java:587]: Should we adapt the if condition, so the structures selector is also displayed when the feature flag is disabled?

`if (!FeatureFlagManagerUtil.isEnabled("LPS-194763") || (FeatureFlagManagerUtil.isEnabled("LPS-194763") && (_journalDisplayContext.getHighlightedDDMStructureId() <= 0)))`

**Review by ruben-pulido — CHANGES_REQUESTED** [review]

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalManagementToolbarDisplayContext.java (hunk 1 above)
Comment (jkappler) [reply modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalManagementToolbarDisplayContext.java]: done

**Comment by ruben-pulido** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward:force

**Pull request closed without merging**